controllers/amphoracontroller_controller.go

```diff
@@ -344,15 +344,21 @@ func (r *OctaviaAmphoraControllerReconciler) reconcileNormal(ctx context.Context
 			},
 		)
 		if err != nil {
+			if k8s_errors.IsNotFound(err) {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.TLSInputReadyCondition,
+					condition.RequestedReason,
+					condition.SeverityInfo,
+					fmt.Sprintf(condition.TLSInputReadyWaitingMessage, instance.Spec.TLS.CaBundleSecretName)))
+				return ctrl.Result{}, nil
```

Comment:
Same here regarding `RequeueAfter`

Reply:
we index/watch the named cert secret, should not be required, or why you think we should have it?